scripts/apply-ci-settings-file.xsh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env xonsh
+
+# Copyright (c) 2025 Toradex
+# SPDX-License-Identifier: MIT
+
+##
+# This script is used to verify the sanity of the CI environment.
+# Is useful to show to the user the env that should be set
+# and fail fast if something is missing.
+##
+
+# use the xonsh environment to update the OS environment
+$UPDATE_OS_ENVIRON = True
+# Get the full log of error
+$XONSH_SHOW_TRACEBACK = True
+# this script should handle the subprocess errors
+$RAISE_SUBPROC_ERROR = False
```

Comment:
Was this done intentionally? The script should not fail if some of the commands fail?